File: sort.py
import time
import numpy

n = 0
a = []
with open("random8.txt", "r") as file:
    n = int(file.readline())
    for x in file.readline().strip().split():
        a.append(int(x))

start = time.time()

b = numpy.sort(a)

end = time.time()

execution_time = end - start

print(execution_time)

File: sort.cpp
#include<bits/stdc++.h>
using namespace std;

#define N 1000000

int n = N, a[N];

int main(){

    freopen("random8.txt", "r", stdin);

    clock_t start = clock();

    cin >> n;
    for(int i = 0 ; i < n; i++) cin >> a[i];

    sort(a, a+n);
 
    clock_t end = clock();

    cout << "Time run: " << (float)(end-start)/CLOCKS_PER_SEC << "s" << "\n";
}